php/libraries/LorisForm.class.inc

```diff
@@ -1892,9 +1892,13 @@ class LorisForm
                 $el['type'] = $type;
             }
             break;
+        case 'date':
+            $el            = $this->createBase($elname, $label, $attribs);
+            $el['type']    = $type;
```

Comment:
isn't `type` the `<input type="..">` attribute?

Why would a date be a type other than "date"? (In fact, it doesn't look like it's possible in this switch statement, so maybe just change it to the string instead of the variable to make this a little clearer.)